data-reporting-service/src/main/resources/db/odse/008-sp_patient_race_event-001.sql

```diff
@@ -1,4 +1,4 @@
-CREATE or ALTER PROCEDURE dbo.sp_patient_race_event @user_id_list nvarchar(max)
+CREATE OR ALTER PROCEDURE dbo.sp_patient_race_event @user_id_list nvarchar(max)
```

Comment:
Since sp_patient_race_event is a sproc called within sp_patient_event, would it be better to call it before?

Reply:
@upasanapattnaik-eq The proc will still get created with warning. And the the race proc will be eventually be created. Not an issue

Reply:
@rseshadri99 @upasanapattnaik-eq anyway, let's make it logical in changelog as well